tests/test_report.py

```diff
@@ -0,0 +1,85 @@
+from unittest.mock import patch
+
+from freezegun import freeze_time
+
+from dsc.report import FinalizeReport
+from dsc.workflows.base import WorkflowEvents
+
+
+@freeze_time("2025-01-01 09:00:00")
+def test_finalize_report_init_success(workflow_events_finalize):
+    finalize_report = FinalizeReport(
+        workflow_name="test", batch_id="aaa", events=workflow_events_finalize
+    )
+
+    assert finalize_report.workflow_name == "TEST"
+    assert finalize_report.batch_id == "aaa"
+    assert finalize_report.report_date == "2025-01-01 09:00:00"
+    assert finalize_report.events == workflow_events_finalize
+
+
+@freeze_time("2025-01-01 09:00:00")
+def test_finalize_report_init_from_workflow_success(
+    base_workflow_instance, workflow_events_finalize
+):
+    finalize_report = FinalizeReport.from_workflow(base_workflow_instance)
+
+    assert finalize_report.workflow_name == base_workflow_instance.workflow_name.upper()
+    assert finalize_report.batch_id == base_workflow_instance.batch_id
+    assert finalize_report.report_date == "2025-01-01 09:00:00"
+    assert isinstance(finalize_report.events, WorkflowEvents)
+
+
+def test_finalize_report_subject_success(workflow_events_finalize):
+    finalize_report = FinalizeReport(
+        workflow_name="test", batch_id="aaa", events=workflow_events_finalize
+    )
+    assert finalize_report.subject == "DSpace Submission Results - TEST, batch='aaa'"
+
+
+def test_finalize_report_status_returns_success(workflow_events_finalize):
+    finalize_report = FinalizeReport(
+        workflow_name="test", batch_id="aaa", events=workflow_events_finalize
+    )
+    assert finalize_report.status == "success"
```

Comment:
Could these tests be combined?

Reply:
Hmm, I could use some clarification:
* `test_finalize_report_subject_success` verifies the **subject heading** set for the email for a 'finalize' report 
* `test_finalize_report_status_returns_success` verifies the **status** included in the email message body for a 'finalize' report (see screenshot below for sample message body).

![image](https://github.com/user-attachments/assets/e04abc05-0161-4790-826a-d58d3945a002)

Reply:
OK, sorry, I misinterpreted what's happening here, totally fine as is, thank you for clarifying!